src/main/resources/persistence.properties

```diff
@@ -1,11 +1,11 @@
-hibernate.dialect = org.hibernate.dialect.PostgreSQL9Dialect
+hibernate.dialect = org.hibernate.spatial.dialect.postgis.PostgisDialect
 hibernate.connection.driver_class = org.postgresql.Driver
 hibernate.archive.autodetection = class
 
-javax.persistence.jdbc.url = jdbc:postgresql://188.166.92.29:5432/moodcat_db
-javax.persistence.jdbc.user = moodcat
-#hibernate.hbm2ddl.auto = create-drop
-hibernate.hbm2ddl.auto = update
+javax.persistence.jdbc.url = jdbc:postgresql://localhost:5432/moodcat
+javax.persistence.jdbc.user = postgres
```

Comment:
Correct, but  I think it´s best too keep it this way for a while so we can test under postgrse locally using a create-drop schema. And then with deployment switch back to the original settings, when the server is prepared with both the new geometry types and the new user enitity.